pkg/operator/controllers/workaround/systemreserved.go

```diff
@@ -0,0 +1,197 @@
+package workaround
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"fmt"
+
+	azureproviderv1beta1 "github.com/openshift/cluster-api-provider-azure/pkg/apis/azureprovider/v1beta1"
+	clusterapi "github.com/openshift/cluster-api/pkg/client/clientset_generated/clientset"
+	mcv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	mcoclient "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/util/retry"
+
+	"github.com/Azure/ARO-RP/pkg/dynamichelper"
+	"github.com/Azure/ARO-RP/pkg/util/version"
+)
+
+type systemreserved struct {
+	mcocli     mcoclient.Interface
+	clustercli clusterapi.Interface
+	dh         dynamichelper.DynamicHelper
+	log        *logrus.Entry
+}
+
+const (
+	labelName                  = "bz-1857446"
+	labelValue                 = "hotfixed"
+	machineConfigName          = "bz-1857446-hotfix"
```

Comment:
Should be KubeletConfig not machineConfig name.

Do we want to put the machineConfigPool name here as well? ("worker")